tests/queries/0_stateless/02932_refreshable_materialized_views_1.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
-# Tags: atomic-database
+# Tags: atomic-database, no-parallel
+# Tags: no-parallel - due to SYSTEM TEST VIEW SET FAKE TIME
```

Comment:
But each table has its own fake time, how does it break parallel tests?

Reply:
Yep, you are right, removed